### PR TITLE
Bug 1855635: Update deps to k8s 1.19-rc.2

### PR DESCRIPTION
**Pull request by damemi**

### CODE REVIEW

**Comment by damemi** [comment]
This is failing with `leaderelection lost`, looking into what could be causing it...

From `artifacts/e2e-aws-operator/installer/bundle/bootstrap/containers/bootstrap-control-plane/kube-controller-manager.log`:
```
F0728 18:36:31.672192       1 controllermanager.go:306] leaderelection lost
goroutine 1 [running]:
k8s.io/kubernetes/vendor/k8s.io/klog/v2.stacks(0xc0007af101, 0xc001f84b40, 0x4c, 0xe1)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/klog/v2/klog.go:996 +0xb8
k8s.io/kubernetes/vendor/k8s.io/klog/v2.(*loggingT).output(0x77b07a0, 0xc000000003, 0x0, 0x0, 0xc002cc0150, 0x7620281, 0x14, 0x132, 0x0)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/klog/v2/klog.go:945 +0x19d
k8s.io/kubernetes/vendor/k8s.io/klog/v2.(*loggingT).printf(0x77b07a0, 0x3, 0x0, 0x0, 0x47bc3ba, 0x13, 0x0, 0x0, 0x0)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/klog/v2/klog.go:733 +0x17a
k8s.io/kubernetes/vendor/k8s.io/klog/v2.Fatalf(...)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/klog/v2/klog.go:1456
k8s.io/kubernetes/cmd/kube-controller-manager/app.Run.func2()
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/cmd/kube-controller-manager/app/controllermanager.go:306 +0x73
k8s.io/kubernetes/vendor/k8s.io/client-go/tools/leaderelection.(*LeaderElector).Run.func1(0xc00020d560)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/client-go/tools/leaderelection/leaderelection.go:200 +0x29
k8s.io/kubernetes/vendor/k8s.io/client-go/tools/leaderelection.(*LeaderElector).Run(0xc00020d560, 0x5016340, 0xc0004a0080)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/client-go/tools/leaderelection/leaderelection.go:210 +0x1a8
k8s.io/kubernetes/vendor/k8s.io/client-go/tools/leaderelection.RunOrDie(0x5016380, 0xc000052060, 0x5057380, 0xc000bfdb80, 0x37e11d600, 0x2540be400, 0xb2d05e00, 0xc00023de40, 0x494c578, 0x0, ...)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/k8s.io/client-go/tools/leaderelection/leaderelection.go:222 +0x96
k8s.io/kubernetes/cmd/kube-controller-manager/app.Run(0xc000702c10, 0xc0000980c0, 0xc000c26e80, 0x0)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/cmd/kube-controller-manager/app/controllermanager.go:298 +0x990
k8s.io/kubernetes/cmd/kube-controller-manager/app.NewControllerManagerCommand.func2(0xc0000d5340, 0xc00015c780, 0x0, 0x6)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/cmd/kube-controller-manager/app/controllermanager.go:133 +0x3d1
k8s.io/kubernetes/vendor/github.com/spf13/cobra.(*Command).execute(0xc0000d5340, 0xc00004c080, 0x6, 0x6, 0xc0000d5340, 0xc00004c080)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/github.com/spf13/cobra/command.go:846 +0x2aa
k8s.io/kubernetes/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc0000d5340, 0x1625fe147bee557a, 0x77b0040, 0xc00006c750)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/github.com/spf13/cobra/command.go:950 +0x349
k8s.io/kubernetes/vendor/github.com/spf13/cobra.(*Command).Execute(...)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/github.com/spf13/cobra/command.go:887
main.main()
	_output/local/go/src/k8s.io/kubernetes/cmd/kube-controller-manager/controller-manager.go:46 +0xcd

```

**Comment by damemi** [comment]
@tnozicka can you approve this pls?

File: go.mod [hunk 1]
```diff
@@ -6,20 +6,20 @@ require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-bindata/go-bindata v3.1.2+incompatible
 	github.com/gonum/graph v0.0.0-20170401004347-50b27dea7ebb
-	github.com/openshift/api v0.0.0-20200701144905-de5b010b2b38
-	github.com/openshift/build-machinery-go v0.0.0-20200512074546-3744767c4131
-	github.com/openshift/client-go v0.0.0-20200521150516-05eb9880269c
-	github.com/openshift/library-go v0.0.0-20200720190023-e1f4b4f4d448
+	github.com/openshift/api master
```
Comment (tnozicka) [inline go.mod]: I don't think this should be separate, just run `go mod tidy && go mod vendor` and commit the result as bump or you are changing the same lines twice in the same PR

Comment (tnozicka) [reply go.mod]: yeah, go mod sucks. but this doesn't give any guarantee, maybe you have amended after this commit and didn't propagate it back. Comment would probably have the same weight, but feel free to leave here.

File: Dockerfile.rhel7 [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.13 AS builder
+FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.14 AS builder
```
Comment (tnozicka) [inline Dockerfile.rhel7:1]: @damemi action item to bump release repo as a followup

**Pull request merged**